src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -1230,10 +1214,38 @@ export class StandaloneSearchInterface extends SearchInterface {
     // By using a setTimeout, we allow other possible code related to the search box / magic box time to complete.
     // eg: onblur of the magic box.
     setTimeout(() => {
-      this._window.location.href = `${link.protocol}//${link.host}${pathname}${link.search}${hash}${HashUtils.encodeValues(stateValues)}`;
+      this._window.location.href = `${link.protocol}//${link.host}${pathname}${link.search}${hash}${this.encodedHashValues}`;
     }, 0);
   }
 
+  private get encodedHashValues() {
+    const values = this.modelAttributesToIncludeInUrl;
+    const uaCausedBy = this.uaCausedBy;
+    const uaMeta = this.usageAnalytics.getCurrentEventMeta();
+
+    if (uaCausedBy != null) {
+      values['firstQueryCause'] = uaCausedBy;
```

Comment:
`values` is a `const`, that we then further modify after creation by adding other properties onto it.  That's misleading.

I'd rather have something like: 
```
const values = {
  ...this. modelAttributesToIncludeInUrl,
  ...this.uaMetadata(), //or any other function name you'd prefer
}
```